package.json

```diff
@@ -33,30 +33,30 @@
     "watch-dev": "gulp watch --bundle"
   },
   "dependencies": {
+    "canvas": "^1.6.7",
```

Comment:
"Canvas" is not needed als dependency because it is only used during development. Should be removed here.

Reply:
Right. I must have placed it there again during testing.